src/main/webapp/resources/js/components/samples/components/SampleInfo.tsx

```diff
@@ -32,7 +34,7 @@ const DEFAULT_HEIGHT = 600;
  */
 export function SampleInfo() {
   const { sample, modifiable: isModifiable } = useSelector(
-    (state) => state.sampleReducer
+    (state: RootStateOrAny) => state.sampleReducer
```

Comment:
Update

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f